### PR TITLE
Gravatar disable dns fix

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes CartoDB/support/issues/840.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -755,8 +755,8 @@ def default_avatar
 
   def gravatar_enabled?
     # Enabled by default, only disabled if specified in the config
-    value = Cartodb.get_config(:avatars, 'gravatar_enabled')
-    value.nil? || value
+    value = Cartodb.config[:avatars] && Cartodb.config[:avatars]['gravatar_enabled']
+    value.to_s != 'false'
```
Comment (mayoral) [inline app/models/user.rb]: These two lines are concise & clear enough to understand the 'enabled by default' commented at the start of the method, so 👍

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -600,6 +600,35 @@ def create_role(user)
...
+      end
+
+      after(:each) do
+        Cartodb::config[:avatars] = @avatars_config
+      end
+
+      it 'should be enabled by default' do
+        user = ::User.new
+        user.gravatar_enabled?.should be_true
+        Cartodb::config[:avatars] = {}
```
Comment (mayoral) [inline spec/models/user_spec.rb]: Nice to test the absence of values, to check the enabled by default behaviour if nothing is defined in the config file.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -600,6 +600,35 @@ def create_role(user)
...
+        user.gravatar_enabled?.should be_true
+        Cartodb::config[:avatars] = { 'gravatar_enabled' => true }
+        user.gravatar_enabled?.should be_true
+        Cartodb::config[:avatars] = { 'gravatar_enabled' => 'true' }
```
Comment (mayoral) [inline spec/models/user_spec.rb]: Maybe we can add a test with a weird string (like `wadus` or `foo`) to show that even with bad inputs the method is going to return true unless the value in the config is `false`).

Comment (juanignaciosl) [inline spec/models/user_spec.rb]: Done!

I've also removed the first test. It hadn't set an explicit configuration to test, so local configuration could break it.

**Comment by juanignaciosl** [comment]
@ernesmb could you please do the acceptance? As you see, the fix is disabling Gravatar (setting it to `false`).

**Comment by ernesmb** [comment]
I have applied the changes to `app/models/user.rb` and it was able to create a new user, so I can confirm the issue is not reproducing anymore.

**Comment by juanignaciosl** [comment]
@ernesmb thanks! To deployment, then! 🚀

**Pull request merged**